### PR TITLE
Gallery demos: reordered and retitled

### PR DESCRIPTION
**Pull request by HansMuller**
Updated per Liam Spradlin's design.

### CODE REVIEW

File: examples/flutter_gallery/lib/gallery/demos.dart [hunk 1]
```diff
@@ -374,63 +374,55 @@ List<GalleryDemo> _buildGalleryDemos() {
     // Cupertino Components
     new GalleryDemo(
       title: 'Activity Indicator',
-      subtitle: 'Cupertino styled activity indicator',
```
Comment (xster) [inline examples/flutter_gallery/lib/gallery/demos.dart]: boo

**Review by xster — APPROVED** [review]
LG

**Comment by cbracken** [comment]
Note: this severely regressed the gallery transition benchmarks.

**Comment by xster** [comment]
I wonder if we're leaking the time of navigating around the categories trying to find the demo into the calculated sum of the transition time costs

**Comment by HansMuller** [comment]
Right, this sounds like a bug in the benchmark since the reordering the lists of demos should have any effect on the time required to launch (the very same) demos.

**Comment by cbracken** [comment]
The benchmarks only count the (avg/max) build time and raster time across all frames, so the total time or number of frames shouldn't significantly matter unless we're running different demos or running more/different transitions than we previously were.

Will see if I can track down what's going on.

**Pull request merged**